superset/translations/zh/LC_MESSAGES/messages.json

```diff
@@ -3330,7 +3330,11 @@
       ],
       "Saved Queries": [
         "已保存查询"
-      ]
+      ],
+      "Prev": ["上一页"],
+      "Next": ["下一页"],
+      "First": ["首页"],
+      "Last": ["尾页"]
```

Comment:
could we move this out into a seperate PR? In general a PR should be a standalone chunk of work